### DISCUSSION TITLE
Unable to Start Interpreter via SSH in Positron

### DISCUSSION THREAD

**Discussion by Inky-River** [posit-dev/positron#5679] [Q&A]
Hello everyone, I’m thrilled to have discovered Positron, such an excellent platform. I’m a complete beginner to Positron, so I’m not sure if the issue I’ve encountered has already been raised or resolved. If that’s the case, I’d like to apologize in advance and thank you all for your patience. 

When using the SSH feature to connect to a remote server, it seems I’m unable to load the interpreter. Under the "Start Interpreter" option, there’s a thick underline, and the console is stuck at "Starting Up" (as shown in the screenshot). I’ve also attached the versions of Positron and my system for reference. I hope someone can help me resolve this. Thank you!
![微信图片_20241210163152](https://github.com/user-attachments/assets/ccf65e7d-19a2-4dc2-9b1a-bb3e9a3bd64a)
![微信图片_20241210163158](https://github.com/user-attachments/assets/b3024cae-493f-480a-a434-a4b825f1d672)
![微信图片_20241210163203](https://github.com/user-attachments/assets/0cc9e25c-2e56-49a0-b699-b6832007baeb)

**Comment by Inky-River** [comment]
I wonder what version of glibc is needed by positron.

**Comment by jennybc** [comment]
I think we are tracking this in #3854. Will you have a look and see if you agree @Inky-River?
